modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/drag-and-drop/DataLayoutBuilderColumnDropZone.es.js

```diff
@@ -130,8 +124,6 @@ export default ({dataLayoutBuilder, node}) => {
 
 	useEffect(() => {
 		dropColumn(node);
-
-		return () => replaceColumn(node);
```

Comment:
The reason I removed this was because `replaceColumn` was removing the node from the nested fields and creating clones the problem with that was that it did an event detach or removing the React link with the node which was causing problems. But I realized later that this corrects errors in the DnD where I should remove a node that was no longer available. I'm wondering if we can approach this in any other way, any suggestions?

Reply:
https://github.com/matuzalemsteles/liferay-portal/pull/37/commits/70c8f48e4a2abccf5568e6f02edd8d102a9789ea I ended up adding some new features that implement `useDrop` on the PageRenderer side.